tests/resources/test_second_quantization.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for functions needed for resource estimation with the double factorization method.
+"""
+import pytest
+
+import pennylane as qml
+
+
+@pytest.mark.parametrize(
+    ("norm", "error", "cost_ref"),
```

Comment:
How is `cost_ref` obtained?

Reply:
This is a general question for the entire test suite.

Reply:
It is computed manually, added a comment to clarify this.